pipeline/run_benchmark.sh

```diff
@@ -34,6 +34,8 @@ OPTIONS:
                      (default: use Harmony binary default)
    -z zone           use the given DNS zone; "" disables DNS-based discovery
                      (default: use Harmony binary default)
+   -t network_type   use the given network type such as mainnet, testnet...
+                     (default: use Harmony binary default)
```

Comment:
default ?

Reply:
“Harmony binary default” – whichever default behavior Harmony binary has, because omitting `-t` here is equivalent to not specifying `-network_type` in benchmark args.